myceliald/src/main.rs

```diff
@@ -57,7 +58,8 @@ async fn run() -> Result<()> {
 
     let storage_handle = storage::new(config.node.storage_path.clone()).await?;
     let runtime_handle = runtime::new(storage_handle.clone());
-    let client_handle = http_client::new(config, runtime_handle, storage_handle);
+    let daemon_storage = daemon_storage::new("./.mycelial_daemon".to_string()).await?; // no idea if this is a good spot to store this info...
```

Comment:
good enough for now. Feels like in near future we can use daemon's database to store this info